aodndata/soop/soop_xbt_dm.py

```diff
@@ -117,8 +117,9 @@ def create_plot(netcdfFilePath, output_dir):
                   lon="%0.2f" % lon,
                   date_start=date_start))
 
-    jpg_output = tempfile.NamedTemporaryFile(delete=False, dir=output_dir)
-    savefig(jpg_output)
+    img_output = tempfile.NamedTemporaryFile(delete=False, dir=output_dir)
+    savefig(img_output)
+    Image.open(img_output.name).convert('RGB').save(img_output.name, 'JPEG', quality=80, optimize=True, progressive=True)  # convert png to jpg
```

Comment:
This seems a bit of a roundabout way of doing it. Not sure what version of matplotlib is required, but the [docs for 3.1.1](https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.pyplot.savefig.html) seem to imply you can just use `savefig` to write a JPG.

This [discussion thread](https://stackoverflow.com/questions/8827016/matplotlib-savefig-in-jpeg-format) (a few years old - see second answer) you still need to install [`pillow`](https://pypi.org/project/Pillow/), which is the updated fork of PIL (which itself is no longer supported, i guess?)

Reply:
yeah this is what I followed

Reply:
putting a new PR. yeah you're right, the new matplotlib version can create a jpg